packages/sdk/src/swap/consts.ts

```diff
@@ -1,4 +1,7 @@
 import { ChainflipNetworks } from '@/shared/enums';
+import { version } from '../../package.json';
```

Comment:
does this get inlined or does it have the entire package.json in the built output

Reply:
i think its inlined - found this in the build: 
<img width="469" alt="Screenshot 2024-07-23 at 10 03 11" src="https://github.com/user-attachments/assets/e5b912af-581a-41e2-b8b7-b760d9730be5">